.azuredevops/modulePipelines/ms.eventgrid.subscriptions.yml

```diff
@@ -0,0 +1,40 @@
+name: 'EventGrid - Subscriptions'
+
+parameters:
+  - name: removeDeployment
+    displayName: Remove deployed module
+    type: boolean
+    default: true
+  - name: prerelease
+    displayName: Publish prerelease module
+    type: boolean
+    default: false
+
+pr: none
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '/.azuredevops/modulePipelines/ms.eventgrid.subscriptions.yml'
+      - '/.azuredevops/pipelineTemplates/*.yml'
+      - '/modules/Microsoft.EventGrid/subscriptions/*'
```

Comment:
Should this be `Microsoft.EventGrid/eventSubscriptions` based on the resource provider? The name should also be reflected in the folder name too

Reply:
I had this orginally so changed it back to `eventSubscriptions`